modules/builds-strategy-pipeline-build.adoc

```diff
@@ -7,6 +7,11 @@
 [id="builds-strategy-pipeline-build_{context}"]
 = Pipeline build
 
+[IMPORTANT]
+====
+The Pipeline build strategy is deprecated in {product-title} 4. Equivalent and better functionality is present in the OpenShift Pipelines based on Tekton.
```

Comment:
Could you also add that Jenkins image on OpenShift are fully supported and users should follow Jenkins instructions for defining their Jenkinsfiles in a job or store it in an SCM: https://jenkins.io/doc/book/pipeline/getting-started/#defining-a-pipeline

Reply:
Done.